src/main/java/io/github/thebusybiscuit/slimefun4/implementation/Slimefun.java

```diff
@@ -258,6 +264,9 @@ private void onUnitTestStart() {
         registry.load(this, config);
         loadTags();
         soundService.reload(false);
+        // TODO: What do we do if tests want to use another storage backend (e.g. testing new feature on legacy + sql)?
+        // Do we have a way to override this?
```

Comment:
I think we should have a way to override this. but not for right now maybe a follow up??

Reply:
agreed